metadata.rb

```diff
@@ -6,9 +6,9 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url 'https://github.com/cvent/octopus-deploy-cookbook'
 issues_url 'https://github.com/cvent/octopus-deploy-cookbook/issues'
-version '0.3.0'
+version '0.4.0'
 
-depends 'windows', '~> 1.38'
+depends 'windows', '~> 1.37'
```

Comment:
Didn't really need to be that strict, really just need windows-service support btu they have made improvements for chef 12 that are nice